src/SteelCap/Bootstrap/TextareaTagHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using SteelCap.Extensions;
+
+namespace SteelCap.Bootstrap
+{
+    [TargetElement("sc-textarea")]
+    public class TextareaTagHelper : TagHelper
+    {
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagName = "textarea";
+
+            output.AppendClass("form-control");
```

Comment:
Lets replace this line with `TextboxCore.AppendClass(output, "form-control");` and remove the AppendClass method from this file.